models/fates/R/write.configs.FATES.R

```diff
@@ -183,14 +196,20 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
 #   ## Write PARAMETER file
    
    ## COPY AND OPEN DEFAULT PARAMETER FILES
+   # TODO: update this to read param files (CLM and FATES) out of the refcase directory, not the PEcAn package
```

Comment:
We've discussed in the past trying to avoid using "TODO's" in the text ... or at least making sure there is an open github issue and putting the link next to the TODO comment

Reply:
Just addressed this @dlebauer